docs/source/user-guide/tasks/docker-image.rst

```diff
@@ -0,0 +1,137 @@
+======================
+Creating Docker Images
+======================
+
+*Available since version XYZ*
+
+Use the ``dockerize`` command to create a Docker image from
+the project. Images created from Anaconda Projects are
+configured to execute a single :doc:`defined command <work-with-commands>`
+in the ``anaconda-project.yml`` file chosen at build time.
+
+The ``dockerize`` command uses `source-to-image (s2i) <https://github.com/openshift/source-to-image#source-to-image-s2i>`_
+to build Docker images using the `s2i-anaconda-project builder images <https://github.com/Anaconda-Platform/s2i-anaconda-project>`_
+that have been uploaded to Docker Hub.
+
+Images built with ``dockerize`` will have a fully prepared ``env_spec`` for the desired
+command and expose port 8086 if the command listens for HTTP requests.
+
+
+Prerequisites
+-------------
+
+In order to utilize the ``dockerize`` command you will need to
+have Docker and source-to-image (s2i) installed.
+
+* `Docker <https://docs.docker.com/get-docker/>`_
+* `source-to-image <https://github.com/openshift/source-to-image#source-to-image-s2i>`_
+
+You can install s2i using Conda
+
+.. code-block:: text
+
+  conda install -c ctools source-to-image
+
+
+Quickstart
+----------
+
+#. Build a docker image to run a supplied command
+
+   .. code-block:: text
+
+     anaconda-project dockerize --command <command-name> -t <image name>
+
+#. Run the Docker image and proxy port 8086
+
+   .. code-block:: text
+
+     docker run -p 8086:8086 <image name>
```

Comment:
Can you explain which is which here (remote or local)? I always get confused with port mapping like this, ending up trying every possible combination until something works.  Would be good to make one of them not be 8086 and to say explicitly which one the user would point their browser to, and which one is being assumed to be provided by the underlying image.